ClLike/cl_like/cl_like.py

```diff
@@ -412,26 +414,73 @@ def _get_cl_data(self, cosmo, **pars):
                         else:
                             cl = ccl.angular_cl(cosmo, t11, t12, ls)
                             cl11[i1, i2, :] = cl
+                        cl11[i1, i2, :] *= clm['pixbeam']
```

Comment:
Hold on, I think we'll be multiplying twice for one of the triangular parts of the auto-correlations, right? I think this should be within the `else`

Reply:
Right, we would be doing the beam multiplication more often than necessary. Better to put it inside the `else`.

Reply:
Not only that, but also one triangle would have the beam multiplied twice, and the other only once